src/css/common.css

```diff
@@ -12,11 +12,14 @@
   --dark-grey: #111111;
 
   --transition: 250ms cubic-bezier(0.4, 0, 0.2, 1);
-  --animation-fast: --container-max-width: 375px;
+  --animation-fast: --container-max-width 0.5s ease-in-out;
```

Comment:
you are using var incorrectly: add `var(--container-max-width) 0.5s ease-in-out`

Reply:
there is a mistake that was added by me -
it should be just a line of code with '--container-max-width: 375px'
it is me who pasted the wrong part '--animation-fast:'

Reply:
ps: Anton will fix it